### ISSUE TITLE
[Other - Accessibility] Focus doesn't move to the expected place. (00.00.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#71223]
#### VFS team name
Veteran Facing Forms

#### VFS product name
Form 21-0966

#### VFS feature name
Intent to file a claim

#### Point of Contact/Reviewers
Brian DeConinck - @briandeconinck - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
No VA.gov Experience Standard for the issue found.

### Issue Details
When starting the form (introduction transitioning to step 1) and ending the form (review page to confirmation page), focus goes to an unexpected place relative to other forms on VA.gov. As a screen reader user, I would expect a consistent experience --- ie. starting a form workflow always sounds like *this* --- and hearing something different would likely cause confusion and cause me to second-guess whether I'm where I think I should be.

Between the two instances where this happens, I think the impact at the start of the form (intro to step 1) is somewhat less, since the user is at the beginning of their workflow. But when losing focus at the end of the form (review to confirmation), it's easy to imagine a confused user thinking their successful submission was unsuccessful, pressing the back button to try to submit again, etc.

It looks like this one is [already documented by Jeana](https://github.com/department-of-veterans-affairs/VA.gov-team-forms/issues/749).

### VA.gov Experience Standard
[Category Number 00, Issue Number 00](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Platform Recommendation
Not sure what makes this form different than others, but focus should land in the expected places as outlined in Jeana's ticket.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by jeana-adhoc** [comment]
[Pull request for first page scrolling issue](https://github.com/department-of-veterans-affairs/vets-website/pull/26980)

**Comment by rhasselle-oddball** [comment]
Intro page bug fixed. Now working on review page bug

**Comment by rhasselle-oddball** [comment]
all seems to be working, addressed from https://app.zenhub.com/workspaces/vagov-product-team-forms-634853151f5c6000165942bc/issues/gh/department-of-veterans-affairs/va.gov-team-forms/661
will need a QA verification pass

**Comment by rhasselle-oddball** [comment]
Working for 10206 which is using our common Confirmation page.
Not working for 996 Confirmation page.
Moving back to Ready for work, to focus on 10206 for now.

**Comment by jeana-adhoc** [comment]
confirmed in staging!
